app/commands/v2/publish.rb

```diff
@@ -177,6 +178,10 @@ def set_publishing_request_id
         )
       end
 
+      def set_update_type
+        edition.update_attributes!(update_type: update_type)
```

Comment:
I'm not sure if this is safe, we could lose minor/major ones to republish. Admittedly it's but a drop in the bucket of our problems on update_type. Nethertheless I'd probably suggest we work on this as part of the [epic](https://trello.com/c/K164C5u8/5-ensure-content-entering-publishing-api-has-an-update-type-of-major-minor)